pkg/kubelet/network/plugins.go

```diff
@@ -38,6 +38,13 @@ type NetworkPlugin interface {
 	// before any other methods are called.
 	Init(host Host) error
 
+	// Tells the plugin to start any long-running operations; the plugin
+	// should not block this function
+	Run() error
+
+	// Indicates to the plugin that the pod CIDR changed, if controlled by kubelet
+	PodCIDRChanged(cidr string)
```

Comment:
Also, the CNI bridge driver errors out if you try to change the bridge's CIDR after it's been created, so if we use CNI (without changes) we can't handle changing PodCIDR anyway.